tools/circt-synth/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(LLVM_LINK_COMPONENTS
+  Support
+)
```

Comment:
If I recall correctly, `Support` is now automatically included for any CIRCT/MLIR tools, such that this is no longer needed.